<launch>
    <node pkg="position_control" type="uwb_control_node" name="uwb_control_node" output="screen">
    <!-- node parameters -->
    <param name="alt_P1" type="double" value= "1.3" />
    <param name="vz_P" type="double" value=  "0.5"/>
    <param name="vz_I" type="double" value=  "0.05"/>
    <param name="vz_D" type="double" value=  "0.2"/>
  
    <param name="xy_P" type="double" value=  "1.0"/>
    <param name="vxy_P" type="double" value= "0.75"/>
    <param name="vxy_I" type="double" value= "0.45"/>
    <param name="vxy_D" type="double" value= "0.11"/>
    <param name="tag_X" type="double" value= "3"/>
    <param name="tag_Y" type="double" value= "3"/>
    <param name="tag_Z" type="double" value= "3"/>

    <param name="max_pitch_roll" type="double" value= "10.0"/>
    </node>
    
</launch>
